docs/source/installation/login.md

```diff
@@ -63,28 +91,32 @@ Click 'Sign in with Keycloak'.
 
 ![Login to Keycloak](../images/keycloak_qhub_login.png)
 
-Enter the username and password you chose when you added a user to QHub above.
+If you chose GitHub or Auth0 login, click the 'GitHub' button to be taken to a GitHub login page and single-sign on from there (as shown in the screenshot above). Otherwise, if you choose 'Password' based authentication, enter the username and password you chose when you added a user to QHub above.
```

Comment:
```suggestion
If you chose GitHub or Auth0 login, click the 'GitHub' button to be taken to a GitHub login page and single-sign-on from there (as shown in the screenshot above). Otherwise, if you choose 'Password-based' authentication, enter the username and password you chose when you added a user to QHub above.
```